### ISSUE TITLE
Model cannot be output if the raise the dimension of word vector

### ISSUE THREAD

**Issue by chuangys** [pommedeterresautee/fastrtext#17]
Everything is okay within the default parameter setting. But when I raised the dimension of word vector too 200, or 300. The model training is still fast but hang at model output. Could you help to check it?

**Comment by pommedeterresautee** [comment]
Hi, can you provide some code?

**Comment by chuangys** [comment]
@pommedeterresautee 
Your code is running well at my environment. So I have to correct my problem.
Apply the same example data, and I use the pre-trained vector, than can reproduce the hang at model output issue.

Source code below:

> library(fastrtext)

data("train_sentences")
data("test_sentences")

# prepare data
tmp_file_model <- tempfile(); print(tmp_file_model);

train_labels <- paste0("__label__", train_sentences[,"class.text"])
train_texts <- tolower(train_sentences[,"text"])
train_to_write <- paste(train_labels, train_texts)
train_tmp_file_txt <- tempfile(); print(train_tmp_file_txt);
writeLines(text = train_to_write, con = train_tmp_file_txt)

test_labels <- paste0("__label__", test_sentences[,"class.text"])
test_texts <- tolower(test_sentences[,"text"])
test_to_write <- paste(test_labels, test_texts)

# learn model
execute(commands = c("supervised", "-input", train_tmp_file_txt,
                     "-output", tmp_file_model, "-dim", 300, "-lr", 1,
                     "-epoch", 300, "-wordNgrams", 2, "-verbose", 1,
                     "-pretrainedVectors", "e:/baproject/data/pretrainedword2vec/wiki-news-300d-1M.vec"))

**Comment by pommedeterresautee** [comment]
it may be related to RAM issue. Did you fixed it?

**Comment by dockstreet** [comment]
Hi - I'm having the same issue as @chuangys, it seems to hang on the larger vec file ? I have 16GB of RAM

**Comment by pommedeterresautee** [comment]
Have you some test code? Did you checked the RAM (model trained by Facebook are quite big).

**Comment by dockstreet** [comment]
I do. 

execute(commands = c("supervised", "-input", "C:/Users/xxx/R/fasttext_test/train.txt",
                     "-output", "C:/Users/xxx/R/fasttext_test/train.bin","-lr", 1,
                     "-epoch", 50,"-wordNgrams", 2, "-verbose", 1 ))

This worked (while the Facebook one would not) : 
https://github.com/jazzyarchitects/fasttext-node/raw/master/train.txt

Here is the RAM size
> memory.limit()
[1] 16204

**Comment by datalee** [comment]
hi，i have a question: the arguments of ' pretrainedVectorsdoes' not support the vec products by [gensim](https://pypi.python.org/pypi/gensim/3.2.0)

**Comment by pommedeterresautee** [comment]
@datalee what is the feature you are referring to?

**Comment by datalee** [comment]
@pommedeterresautee  classification.

**Comment by pommedeterresautee** [comment]
`pretrainedVectors` is the text file produced by fasttext when you learn a model, whatever it is. I don't know the format of gensim but should not be hard to convert (word\tvector where each value is separated by a space).
